pyproject.toml

```diff
@@ -79,6 +79,7 @@ dev-fmt = [
     "pre-commit>=3.0.4,<4.0",
     "pylint>=2.16.2,<3.0",
     "pydeps==1.12.10",
+    "requests==2.31.0"
```

Comment:
any reason for having this version here instead of `requests>=2.28.2,<3",` in `dev-test`?

Reply:
it's just needed for linting now, I can go ahead and throw `dev-test` into the bucket of extras for linting

Reply:
(though this bit is from the branch PRed in #327)